src/main/java/ir/sahab/kafkaconsumer/OffsetTracker.java

```diff
@@ -119,51 +120,70 @@ public OptionalLong ack(int partition, long offset) {
     private class PartitionTracker {
         private final Map<Long /*page index*/, PageTracker> pageTrackers = new HashMap<>();
         private final SortedSet<Long /*page index*/> completedPages = new TreeSet<>();
-        private long lastConsecutivePageIndex;
+        private long lastCompletedPageIndex;
```

Comment:
```suggestion
        private long lastConsecutivePageIndex;
```
Revert the name. This is not last completed page index. This is the last consecutive index in the list of completed indices.

Reply:
Done